mpa/modules/hooks/model_ema_v2_hook.py

```diff
@@ -34,22 +34,26 @@ class ModelEmaV2Hook(Hook):
             to update ema parameters. Defaults to 5.
     """
 
-    def __init__(self, ema_decay=0.999, interval=1, start_epoch=1, **kwargs):
+    def __init__(self, ema_decay=0.9995, interval=1, start_epoch=1, switching_criteria=2000, **kwargs):
```

Comment:
Please add the new parameter to the docstring. Also, the name `switching_criteria` sounds a bit unclear: 
something like `data_len_threshold` reveals the role of this arg.